spec/requests/v1/pacbio/runs_spec.rb

```diff
@@ -1382,12 +1402,18 @@
                   id: plate.id,
                   wells_attributes: [
                     {
-                      id: plate.wells.first.id.to_s,
-                      pool_ids: [plate.wells.first.pools[0].id, pool1.id]
+                      id: well1.id.to_s,
```

Comment:
I think we also want to check adding an aliquot with the same tag as a _destroy aliquot to test the validation doesn't break.